eva/binder/statement_binder_context.py

```diff
@@ -91,6 +91,17 @@ def _check_derived_table_alias_map(self,
             for obj in col_objs:
                 if obj.name.lower() == col_name:
                     return obj
+    
+    def _search_all_col_name(self, alias: str) -> List[str]:
+        table_obj = self._table_alias_map.get(alias, None)
+        if table_obj:
+            return [col.name for col in table_obj.columns]
+        col_objs = self._derived_table_alias_map.get(alias, None)
+        if col_objs:
+            return [obj.name for obj in col_objs]
+        err_msg = 'Cannot find alias = {}'.format(alias)
+        logger.error(err_msg)
```

Comment:
@jarulraj With the new logger, does the `logger.exception(error)` also raise the exception (so that we can merge lines 103-104)?

Reply:
Not sure. We could add that functionality.

Reply:
I don't think so. `logger.exception(err_msg)` is equivalent to `logger.error(err_msg, exc_info=True)`